src/helpers/useUndo.ts

```diff
@@ -190,7 +190,7 @@ export const useUndo = ({
       description: 'Redo changes',
       callback: (event) => {
         event.preventDefault();
-        if (!disabled) {
+        if (!disabled && canRedo) {
```

Comment:
@amcdnl Not sure about that, please confirm.

Reply:
Good catch!